mountpoint-s3/src/fs.rs

```diff
@@ -1156,29 +1146,11 @@ where
         let mut state = file_handle.state.lock().await;
         match &mut *state {
             FileHandleState::Created { .. } => Ok(()),
-            FileHandleState::Read { pid: open_pid, .. } => {
```

Comment:
is the comment on line 1135 still relevant?
```
// The same for read path. We want to stop the prefetcher and decrease the reader count
// as soon as users close a file descriptor so that we don't block users from doing other
// operation like overwrite the file.
```

Reply:
good spot